flask/files/README.md

```diff
@@ -0,0 +1,24 @@
+# Platform.sh Python 2.7 minimal example
```

Comment:
This file looks like a copy-paste from the minimal Python repo.  It should be updated to say that it's Flask.

Reply:
PR every time.  The goal is to hook those PRs up to Psh test environments so we can review the changes before merging them.